### PR TITLE
place server on the same URL so that it can easily served by ngrok

### PR DESCRIPTION
**Pull request by sebastianovide**
it also simplifies the getting started as it requires just `npm install && npm start`

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+app.use(express.static(path.join(__dirname)));
+
+// Proxy handler
+const proxyHandler = async (req, res) => {
+  try {
+    const targetUrl = `https://tiles.soundscape.services${req.originalUrl}`;
+
+    const response = await axios.get(targetUrl, {
+      headers: {
+        "User-Agent": "YourNewUserAgent",
```
Comment (steinbro) [inline server.js:23]: Should probably set this to something meaningful.

File: server.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  }
+};
+
+// Route for /tiles
+app.use("/tiles/*", proxyHandler);
+
+// Fallback to serve the index.html for single-page applications
+app.get("*", (req, res) => {
+  res.sendFile(path.join(__dirname, "index.html"));
+});
```
Comment (steinbro) [inline server.js:46]: Doesn't seem necessary given lines 13-14 serving static files.

File: package.json [hunk 3]
```diff
@@ -1,12 +1,16 @@
 {
   "dependencies": {
     "@turf/turf": "^6.5.0",
+    "axios": "^1.7.2",
     "chai": "^5.1.0",
+    "express": "^4.19.2",
+    "http-proxy-middleware": "^3.0.0",
```
Comment (steinbro) [inline package.json:7]: This looks like a package you considered but didn't end up using, right?

**Review by steinbro — COMMENTED** [review]
Ah, thank you! I very much like consolidating to a single server, a single language, and a single README step. Just a few trivial comments, which I'll address and merge.

File: server.js (hunk 1 above)
Comment (sebastianovide) [reply server.js:23]: yes... you are right... (I confess that I didn't spend much time on it... and ChatGPT helped ;) )

**Pull request merged**